src/Validator/DuplicateKeysValidator.php

```
@@ -35,46 +31,22 @@ public function processFile(ParserInterface $file): array
     }
 
-    /**
-     * @param array<string, array<int, array{
-     *      file: string,
-     *      issues: array<string, int>,
-     *      parser: string,
-     *      type: string
-     *  }>> $issueSets
-     */
-    public function renderIssueSets(InputInterface $input, OutputInterface $output, array $issueSets): void
+    public function formatIssueMessage(Issue $issue, string $prefix = '', bool $isVerbose = false): string
```

PHPMD (2.15.0)

Avoid unused parameters such as '$isVerbose'. (Unused Code Rules)

(UnusedFormalParameter)